actors/builtin/miner/monies.go

```diff
@@ -9,36 +9,37 @@ import (
 )
 
 // IP = IPBase(precommit time) + AdditionalIP(precommit time)
-// IPBase(t) = InitialPledgeFactor * BR(t)
+// IPBase(t) = BR(t, InitialPledgeProjectionPeriod)
 // AdditionalIP(t) = LockTarget(t)*PledgeShare(t)
 // LockTarget = (LockTargetFactorNum / LockTargetFactorDenom) * FILCirculatingSupply(t)
 // PledgeShare(t) = sectorQAPower / max(BaselinePower(t), NetworkQAPower(t))
 // PARAM_FINISH
-var PreCommitDepositFactor = big.NewInt(20)
-var InitialPledgeFactor = big.NewInt(20)
+var PreCommitDepositFactor = 20
+var InitialPledgeFactor = 20
+var PreCommitDepositProjectionPeriod = abi.ChainEpoch(PreCommitDepositFactor) * builtin.EpochsInDay
+var InitialPledgeProjectionPeriod = abi.ChainEpoch(InitialPledgeFactor) * builtin.EpochsInDay
 var LockTargetFactorNum = big.NewInt(3)
 var LockTargetFactorDenom = big.NewInt(10)
 
-// FF = (DeclaredFaultFactorNum / DeclaredFaultFactorDenom) * BR(t)
-var DeclaredFaultFactorNum = big.NewInt(214)
-var DeclaredFaultFactorDenom = big.NewInt(100)
+// FF = BR(t, DeclaredFaultProjectionPeriod)
+// projection period of 2.14 days  2880 * 2.14 = 6163.2.  Rounded to nearest epoch 6163
+var DeclaredFaultProjectionPeriod = abi.ChainEpoch(6163)
```

Comment:
Which epoch duration is this calculated with? Could you do the arithmetic with integers (e.g. scaled by 10000) so it stays current?